src/AppV2.js

```diff
@@ -49,6 +56,12 @@ const App = () => {
           <Suspense fallback={<div>Loading...</div>}>
             <Switch>
               <Route exact path="/rc" component={Dashboard} />
```

Comment:
Question for @manuel-calavera : if we don't include the data stores in App.js, but they are included in the individual components, does that mean we are doing re-initialization in every route?

My intuition is that the stores should be created & initialized in App.js so that the data is persistent/more responsive, but maybe that's already the case in the current model?

Reply:
Store is being created and initialized only once. App.js should not have any logic (besides routing), should be small and easy to read.